prac_05/emails.py

```diff
@@ -0,0 +1,32 @@
+"""
+CP1404 Practical 05 Hunter Kruger-Ilingworth
+Estimate: 30 mins
+Actual: 37 mins
+"""
+
+
+def main():
+    name_to_email = {}
+    pair = get_pair()
+    while get_pair() != "":
```

Comment:
if you don't place the while loop within the function, then replace the get_pair() here with the pair variable, as here the user is asked twice in a row, with the first time deciding what pair will be and the second time deciding if the while loop will terminate, rather than one question that answers both if that makes sense